App/src/test/java/cc/softwarefactory/lokki/android/espresso/GeneralNavigationTest.java

```diff
@@ -0,0 +1,82 @@
+package cc.softwarefactory.lokki.android.espresso;
+
+import cc.softwarefactory.lokki.android.R;
+import cc.softwarefactory.lokki.android.espresso.utilities.MockJsonUtils;
+import cc.softwarefactory.lokki.android.espresso.utilities.TestUtils;
+import com.squareup.okhttp.mockwebserver.MockResponse;
+
+import org.json.JSONException;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.pressBack;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.pressMenuKey;
+import static android.support.test.espresso.action.ViewActions.swipeLeft;
+import static android.support.test.espresso.action.ViewActions.swipeRight;
+import static android.support.test.espresso.assertion.ViewAssertions.doesNotExist;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.hasSibling;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.isRoot;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.not;
+
+import android.support.test.espresso.action.GeneralLocation;
+import android.support.test.espresso.action.GeneralSwipeAction;
+import android.support.test.espresso.action.Swipe;
+import android.support.test.espresso.action.ViewActions;
+import android.util.Log;
+/**
+ * Created by Little-Black on 2/1/15.
+ */
+public class GeneralNavigationTest extends LoggedInBaseTest{
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+    }
+
+
+//TEST
+    public void testClickDecorToShowNavigationDrawer() {
+        getActivity();
+        onView(withId(R.id.decor_content_parent)).perform(TestUtils.clickScreenPosition(0, 0));
+        onView(allOf(withText(R.string.app_name), hasSibling(withText(R.string.contacts)))).check(matches(isDisplayed()));
+        onView(withText(R.string.contacts)).check(matches(isDisplayed()));
+        onView(withText(R.string.settings)).check(matches(isDisplayed()));
+        onView(withText(R.string.about)).check(matches(isDisplayed()));
+    }
+
+    public void testClickMenuKeyToShowNavigationDrawer()  {
+        getActivity();
+        onView(isRoot()).perform(pressMenuKey());
+        onView(allOf(withText(R.string.app_name), hasSibling(withText(R.string.contacts)))).check(matches(isDisplayed()));
+        onView(withText(R.string.contacts)).check(matches(isDisplayed()));
```

Comment:
Or rather than check if there's a Lokki text which has contacts text as a sibling, is it possible to check for Lokki name that is inside the navigation?